.github/workflows/check-migrations.yml

```diff
@@ -0,0 +1,35 @@
+name: Check Migrations on Paseo
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+  workflow_dispatch:
+    inputs:
+      runtime-package:
```

Comment:
Do we have anything other than frequency-runtime?

Reply:
We only got frequency-runtime as our main substrate runtime code

Reply:
I have hardcoded `frequency-runtime` since we only got one for now